test/libethereum/BlockchainTestsFiller/bcValidBlockTestFiller.json

```diff
@@ -889,7 +889,7 @@
         ]
     },
 
-    "ExtraData1024" : {
+    "ExtraData32" : {
         "genesisBlockHeader" : {
```

Comment:
this test was moved to invalid. so this particular one is a new valid test with extraData32